WordPress/gutenberg
# [RNMobile] Fix toolbar above the keyboard

## Description

Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/2592

After merging https://github.com/WordPress/gutenberg/pull/24414, I've noticed that toolbar is hanging in case when user is adding a new block from previously focused `Paragraph`



Correct usage of `SafeArea` in `KeyboardAvoidingView`

## How has this been tested?

* Open mobile app
* Press FAB button
* Choose Site page
* **Expect toolbar should be located above the keyboard**
* Rotate the device to landscape
* **Expect toolbar should be located above the keyboard**
* Rotate the device to horizontal
* Focus `Paragraph`  block
* Press inserter and choose block e.g. `Image`
* **Expect toolbar should be located at the bottom of the screen and mustn't hang**
* Rotate the device to landscape
* **Expect toolbar should be located at the bottom of the screen and mustn't hang**
* Focus the `Paragraph` 
* **Expect toolbar should be located above the keyboard**
* Rotate the device to horizontal
* **Expect toolbar should be located above the keyboard**





## Screenshots 

iPhone 11 | iPhone 8
--- | ---
![ip11](https://user-images.githubusercontent.com/22746080/90253175-079a1380-de41-11ea-8a3f-7fd34a309bb5.gif) | ![ip8](https://user-images.githubusercontent.com/22746080/90253182-0b2d9a80-de41-11ea-9a1c-57c500c0f39c.gif)


## Types of changes





## Checklist:
